client/flags/flags.go

```diff
@@ -61,6 +61,9 @@ const (
 	FlagPage             = "page"
```

Comment:
should we remove this one?

Reply:
May be yes after all migrations done.

Reply:
yeah, agreed with @atheeshp: auth,ibc,evidence,gov etc are all using FlagPage, probably should wait for those modules to switch to using gRPC & query.PageRequest first, or else this PR will become too big.